uisdk/src/main/java/com/karhoo/uisdk/screen/booking/domain/support/ContactEmailProvider.kt

```diff
@@ -0,0 +1,90 @@
+package com.karhoo.uisdk.screen.booking.domain.support
+
+import android.app.Activity
+import android.content.Intent
+import android.net.Uri.parse
+import com.karhoo.sdk.api.KarhooApi
+import com.karhoo.sdk.api.model.TripInfo
+import com.karhoo.uisdk.R
+import com.karhoo.uisdk.util.VersionUtil
+import java.lang.ref.WeakReference
+
+class ContactEmailProvider(activity: Activity) {
```

Comment:
Do we need any tests for this?

Reply:
testing this part seems to be a bit challenging due to some parts are Static method/classes and not very easy mockable .... will tackle the tests separately to avoid this branch to be stale

Reply:
Managed to add tests. Thanks for your help

First, lets merge https://github.com/karhoo/karhoo-android-ui-sdk/pull/74 and then we can merge this one